app/assets/stylesheets/local/users/_users.scss

```diff
@@ -35,44 +35,27 @@ form.user_form {
   }
 
   div.form-field#assignments {
-
-    div.item {
-      padding: 2px 0;
+    .mission,
+    .role {
+      display: inline-block;
+      margin-right: 10px;
+      width: 150px;
     }
 
-    div.assignment, div.assignment_header {
-      display: inline-block;
-      min-height: 20px;
+    .assignment-row {
+      height: 50px;
```

Comment:
Please don't use hard coded heights! Does not work well with people changing font sizes, etc.